SQL/migrate/V023__newscaster_static_news.sql

```diff
@@ -0,0 +1,35 @@
+--
+-- Adds a new table to load news from
+--
+CREATE TABLE `ss13_news_channels` (
+	`id` INT(11) NOT NULL AUTO_INCREMENT,
+	`name` VARCHAR(50) NOT NULL,
+	`author` VARCHAR(50) NOT NULL,
+	`locked` TINYINT(3) UNSIGNED NOT NULL DEFAULT '1',
+	`is_admin_channel` TINYINT(3) UNSIGNED NOT NULL DEFAULT '1',
+	`announcement` VARCHAR(200) NOT NULL,
+	`created_by` VARCHAR(50) NOT NULL,
+	`created_at` DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+	`updated_at` DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+	`deleted_at` DATETIME NULL DEFAULT NULL,
+	PRIMARY KEY (`id`)
+)
+ENGINE=InnoDB;
+
+CREATE TABLE `ss13_news_stories` (
+	`id` INT(11) NOT NULL AUTO_INCREMENT,
+	`channel_id` INT(11) NOT NULL,
+	`author` VARCHAR(50) NOT NULL,
+	`body` TEXT NOT NULL,
+	`message_type` VARCHAR(50) NOT NULL DEFAULT 'Story',
```

Comment:
ENUM perhaps?

Reply:
Intentionally a varchar, because there is no need to restrict it to a set of pre-defined message types.
(The text thats entered there is displayed at the newscasters as it is)